.vscode/settings.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
in my experience this is actually valuable since it sets coding standards - its one of the big benefits of workspace settings in vs code

we do it in all of our projects on @getsentry

Reply:
Then it's probably a good idea to include it. Haven't worked with VS Code that much yet – I'm  using the JetBrains apps most of the time...